### ISSUE TITLE
Support .NET Core 3 WPF

### ISSUE THREAD

**Issue by DaveNay** [XAMLMarkupExtensions/WPFLocalizationExtension#181]
Are there plans for adding support for .NET Core 3.0 WPF?

I am willing to help with this effort if needed, although I really don't know where to start other than adding `netcoreapp3.0` to the `<TargetFrameworks></TargetFrameworks>` element.

**Comment by konne** [comment]
I really like .net core and I heavily use it. I will have a look for .NET Core 3.0 after the release.
XAMLMarkupExtensions and WPFLocalizationExtension are already ported to the new csproj format, so just adding .NET Core 3.0 and some conditional compiles for some packagereferences would be no issue.
I will let this issue open, but please ping me again after .NET Core 3.0 is released and I will check it out.

**Comment by mldYM** [comment]
when you release .net core 3 supported version on nuget ?

**Comment by ezhassen** [comment]
I added the support (.Net Core 3.1) and  working fine. try it #211 😉 

[WPFLocalizeExtensionEzzFork ](https://www.nuget.org/packages/WPFLocalizeExtensionEzzFork/)

**Comment by konne** [comment]
@DaveNay  please test the current nuget preRelease: 3.4.0-alpha0053
if there are no issues I will make next week the release

**Comment by Jeverson28** [comment]
I've been using 3.4.0-alpha0053 in a .Net Core 3.0 application. My main exe assembly is referencing another assembly that stores the localised string resources used in the application. There is also a neutral language resource embedded in that referenced assembly. When I request a resource I get a System.IO.FileNotFoundException for every culture in .Net Core I don't support. Since I only support 8 languages I receive a lot of exceptions which has dramitically slowed the application's startup (several minutes long). I don't know if there's something wrong with my configuration or if there's an issue with net core 3.x but it does not happen with .Net Framework (I've only tested v4.8 though). 

The exception seems to occur on the line `var rs = resManager.GetResourceSet(c, true, false)` in ResxLocalizationProviderBase.

**Comment by konne** [comment]
@Jeverson28: please create a simple Demo App, or take one of the existing Demo Apps and test it with your issue. If this still exists, please add a new Issue and point on that new PR with Sample or that existing sample

**Comment by Jeverson28** [comment]
I've managed to fix the issue I was having. I thought I'd post it here in case anyone else ever has the same issue.

My application was using Ninject as our DI tool using a "binding by convention" style i.e. bind all classes to appropriate scopes using their suffix. For example I would bind all classes with names that end in ViewModel to their default interface in transient scope. I filtered which assemblies were scanned based on their prefix (all assemblies starting with "CompanyName"). Because all my resource assemblies started with this prefix (e.g. "CompanyName"."ApplicationName".Resources) they were also scanned. For some reason in .Net Core this seems to cause issues when getting the ResourceSet.
